geode-core/src/main/java/org/apache/geode/security/ResourcePermission.java

```diff
@@ -14,6 +14,9 @@
  */
 package org.apache.geode.security;
 
+import java.util.List;
+import java.util.Set;
+
 import org.apache.commons.lang.StringUtils;
 import org.apache.geode.cache.Region;
 import org.apache.geode.management.internal.security.ResourceOperation;
```

Comment:
Both a nitpick and the fault of someone else, but the following files have unused or misordered imports:

```
geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExecuteFunctionCommand.java
geode-core/src/main/java/org/apache/geode/security/ResourcePermission.java
geode-core/src/test/java/org/apache/geode/internal/cache/OplogFlushTest.java
geode-core/src/test/java/org/apache/geode/internal/cache/OverflowOplogFlushTest.java
geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommandTest.java
geode-core/src/test/java/org/apache/geode/management/internal/security/SecurityServiceWithShiroIniIntegrationTest.java
```